frontend/src/pages/[lang]/index.tsx

```diff
@@ -43,7 +43,7 @@ export const HomePage = () => {
       <FormattedMessage id="home.title">
         {(text) => <MetaTags title={text} />}
       </FormattedMessage>
-      <HomepageHero hideBuyTickets={true} />
+      <HomepageHero hideBuyTickets={false} />
```

Comment:
Can we add an env variable a flag maybe later on? Would be easier to hide/show this kind of stuff??

Reply:
we can add a deadline of type tickets! that would be better

Reply:
Uh yeah! we should add a ticket for this !